### PR TITLE
Standardize 2D->2d and 3D->3d in comments.

### PR DESCRIPTION
**Pull request by bangerth**
This is the patch corresponding to #5050, just for comments.

/rebuild

### CODE REVIEW

File: source/initial_temperature/spherical_shell.cc [hunk 1]
```diff
@@ -50,7 +50,7 @@ namespace aspect
          region in between depending on the angle phi=atan2(x,y).
 
          For a plot, see
-         http://www.wolframalpha.com/input/?i=plot+%28%282*sqrt%28x^2%2By^2%29-1%29%2B0.2*%282*sqrt%28x^2%2By^2%29-1%29*%281-%282*sqrt%28x^2%2By^2%29-1%29%29*sin%286*atan2%28x%2Cy%29%29%29%2C+x%3D-1+to+1%2C+y%3D-1+to+1
+         http://www.wolframalpha.com/input/?i=plot+%28%282*sqrt%28x^2%2By^2%29-1%29%2B0.2*%282*sqrt%28x^2%2By^2%29-1%29*%281-%282*sqrt%28x^2%2By^2%29-1%29%29*sin%286*atan2%28x%2Cy%29%29%29%2C+x%3D-1+to+1%2C+y%3d-1+to+1
```
Comment (tjhei) [inline source/initial_temperature/spherical_shell.cc:53]: Can you revert this one?

**Review by tjhei — APPROVED** [review]

File: source/geometry_model/spherical_shell.cc [hunk 2]
```diff
@@ -106,7 +106,7 @@ namespace aspect
               Assert(custom_mesh == slices || custom_mesh == list, ExcNotImplemented());
               // If we are using a custom mesh scheme, we need to create
               // a new triangulation to extrude (this will be a 1D line in
-              // 2D space, or a 2D surface in 3D space).
+              // 2D space, or a 2D surface in 3d space).
```
Comment (gassmoeller) [inline source/geometry_model/spherical_shell.cc:109]: why is this 2D not changed? Did you only substitute one occurrence per line?

**Comment by gassmoeller** [comment]
Personally I prefer 2D and 3D, because 2d and 3d have too many closely related terms or meanings (`2/3 times d` in an equation, or `2nd/3rd` which looks similar), but I couldnt find any agreement on the topic in other projects or dictionaries, and deal.II seems to have roughly the same split (more 2d/3d than 2D/3D), so I guess I am fine with the change. 
Please address Timo's comment and take another look if there are more lines with multiple occurrences (see my comment).

**Comment by bangerth** [comment]
I made the change requested by @tjhei 

@gassmoeller The script didn't convert multiple occurrences on the same line because it was greedy in reading all the way to the last occurrence of 2D/3D. But running the script multiple times fixes this, and that's what I ended up doing.

**Review by tjhei — APPROVED** [review]

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**